client/src/charts/wrapped_nivo/WrappedNivoCircleProportion/WrappedNivoCircleProportion.js

```diff
@@ -119,101 +47,84 @@ export class WrappedNivoCircleProportion extends React.Component {
     const color_scale = scaleOrdinal().range(newIBCategoryColors);
     const value_formatter = get_formatter(is_money, formatter, true, false);
 
-    const graph_data = {
-      id: parent_name,
-      name: parent_name,
-      // ... nivo bubble will roll child values back up and add them to the parent value for use in the graph
-      // so we need to remove the inner portion from the total here
-      value: parent_value - child_value,
-      color: color_scale(parent_name),
-      children: child_value
-        ? [
-            {
-              id: child_name,
-              name: child_name,
-              value: child_value,
-              color: color_scale(child_name),
-            },
-          ]
-        : 0,
+    const Circles = () => {
+      // arbitrary parent values
+      const parent_radius = 20;
+      const parent_cx = 175;
+      const parent_cy = 25;
+
+      // child circle calculations
+      const child_percent = child_value / parent_value;
+      const child_radius = parent_radius * child_percent;
+
+      return (
+        <svg viewBox="0 0 350 350">
```

Comment:
The SVG view box dimensions define the coordinate system inside of itself (to decouple it from the actual HTML size of the svg element itself). I'd suggest only hardcoding the viewbox length/height value (350) and calculating the parent radius, x, and y from that, so that there's no risk of them getting de-synced later.

Reply:
Done!